embark-ui/src/components/PageHead.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Helmet} from 'react-helmet';
+import {PAGE_TITLE_PREFIX} from '../constants';
+
+const PageHead = ({title, description, enabled = true}) => {
```

Comment:
Wonder if we should rename this to something more accurate like `MetaData` or `MetaContainer` or something like that, so that it's indicated that this deals primarily with meta data. 

Right now it's just title and description, but maybe we'll add other things in the future (social graph stuff etc. This is especially useful when Cockpit becomes a hosted platform)